src/services/services.ts

```diff
@@ -1750,6 +1750,7 @@ namespace ts {
             synchronizeHostData();
             const sourceFile = getValidSourceFile(fileName);
             const node = getAdjustedRenameLocation(getTouchingPropertyName(sourceFile, position));
+            if (!Rename.nodeIsEligibleForRename(node)) return undefined;

```

Comment:
1. this isn't required for this fix, but seems consistent with this change - not computing locations we're not going to use.
2. it does mean though that  for valid attempts we're checking for `nodeIsEligibleForRename` twice: first in `mapRenameInfo -> getRenameInfo` and then in `combineProjectOutputForRenameLocations -> combineProjectOutputWorker -> findRenameLocations`.